src/translation_table.py

```diff
@@ -0,0 +1,267 @@
+from bs4 import BeautifulSoup
+
+
+EN = (
+    {
+        'variable': 'congressperson_name',
+        'name': 'Congressperson Name',
+        'desc': """Name used by the congress person during his term in
+            office. Usually it is composed by two elements: a given name and a
+            family name; two given names; or two forename, except if the head
+            of the Lower House explicitly alter this rule in order to avoid
```

Comment:
"Câmara dos Deputados" refer themselves as "Chamber of Deputies" in [their English version](http://www2.camara.leg.br/english). Shouldn't we prefer this other name?